package.json

```diff
@@ -150,15 +150,15 @@
     "@nestjs/cli": "^9.4.0",
     "@nestjs/schematics": "^10.1.1",
     "@nestjs/testing": "^10.3.2",
-    "@nx/cypress": "^17.1.3",
-    "@nx/eslint-plugin": "^16.9.1",
-    "@nx/jest": "^16.9.1",
-    "@nx/js": "^16.9.1",
-    "@nx/next": "^16.9.1",
-    "@nx/node": "^18.0.2",
-    "@nx/react": "^16.9.1",
-    "@nx/storybook": "^17.1.3",
-    "@nx/workspace": "^16.9.1",
+    "@nx/cypress": "18.0.3",
```

Comment:
Same comment as the other PR (which is still probably out of scope here) but I find mentions of [`@nx/cypress`](https://nx.dev/nx-api/cypress) suspicious and we might be able to ✂️ .

Although now I almost definitely feel like we're out of scope and we should file a Jira and cite https://nx.dev/nx-api/playwright (plus, like, figure out what that plugin does that we may not get with our current Playwright setup).

Reply:
I mentioned it in my comment on the other PR, but I'm pretty sure `cypress` was added by accident when we initially added Next.js and forgot to specify no e2e testing library.

So I've just removed it as part of this PR, and seems to be fine.